### ISSUE TITLE
does this work with other modlet

### ISSUE THREAD

**Issue by Anuj79** [dave-atx/homebridge-platform-smartac#4]
does this work with other modlet only which doesnt have a thermostat
https://shop.thinkecoinc.com/products/wifi-modlet-bn#.Wei7ZBOPKu4

**Comment by dave-atx** [comment]
It doesn't work with non-thermostat modlets, but I'd be happy to accept a PR that adds support for them.

**Comment by Anuj79** [comment]
i can give it a go.. it would seem its simpler than a thermostat as it's just on and off..

**Comment by dave-atx** [comment]
Cool, yeah I tend to agree. Assuming those are controlled by mymodlet.com, it's probably a small amount of work. I don't have one to test, but I'd be happy to offer comments on a PR & merge when it's ready.

**Comment by Anuj79** [comment]
yeah i saw the code and see you were using mymodlet.. i think it should work.

**Comment by Anuj79** [comment]
i ran this link  from your code after logging in https://mymodlet.com/SmartAC/UserSettingsTable and seems like its only showing Thermostats and not Dehumidifies(stand alone switch)

did some network capture from the app and saw they were using restful api and Oauth so i could possible use that.. 
because it will be oauth to log in and then using token to set the switch on and off..

**Comment by Anuj79** [comment]
i did modify it to support Modlet.. but i had to fork it as i couldnt modify it as i didnt have themorstat..
